kong/content.md

```diff
@@ -0,0 +1,91 @@
+# What is Kong?
+
+Kong was built to secure, manage and extend Microservices & APIs. If you're building for web, mobile or IoT (Internet of Things) you will likely end up needing to implement common functionality on top of your actual software. Kong can help by acting as a gateway for any HTTP resource while providing logging, authentication and other functionality through plugins.
+
+Powered by NGINX and Cassandra with a focus on high performance and reliability, Kong runs in production at Mashape where it has handled billions of API requests for over ten thousand APIs.
+
+Kong's documentation can be found at [getkong.org/docs](http://getkong.org/docs).
+
+%%LOGO%%
+
+# How to use this image
+
+First, Kong requires a running Cassandra or PostgreSQL cluster before it starts. You can either use the official Cassandra/PostgreSQL containers, or use your own.
+
+## 1. Link Kong to either a Cassandra or PostgreSQL container
+
+It's up to you to decide which datastore between Cassandra or PostgreSQL you want to use, since Kong supports both.
+
+### Cassandra
+
+Start a Cassandra container by executing:
+
+```shell
+$ docker run -d --name kong-database \
+                -p 9042:9042 \
+                cassandra:2.2
+```
+
+### Postgres
+
+Start a PostgreSQL container by executing:
+
+```shell
+docker run -d --name kong-database \
+                -p 5432:5432 \
+                -e "POSTGRES_USER=kong" \
+                -e "POSTGRES_DB=kong" \
+                postgres:9.4
+```
+
+### Start Kong
+
+Once the database is running, we can start a Kong container and link it to the database container, and configuring the `DATABASE` environment variable with either `cassandra` or `postgres` depending on which database you decided to use:
+
+```shell
+$ docker run -d --name kong \
+    -e "DATABASE=cassandra" \
+    --link kong-database:kong-database \
+    -p 8000:8000 \
+    -p 8443:8443 \
+    -p 8001:8001 \
+    -p 7946:7946 \
+    -p 7946:7946/udp \
+    --security-opt seccomp:unconfined \
+    mashape/kong
```

Comment:
This should just be `kong` :wink:

Is `--security-opt seccomp:unconfined` actually required?  You've got a note explaining what to do if it errors out, but nothing here explaining why it's necessary or even recommended, which is much more interesting IMO. :smile: